netspresso/compressor/v2/compressor.py

```diff
@@ -0,0 +1,540 @@
+from pathlib import Path
+import sys
+from typing import Dict, List, Optional
+from urllib import request
+
+from loguru import logger
+
+from netspresso.clients.auth import TokenHandler, auth_client
+from netspresso.clients.compressor import compressor_client_v2
+from netspresso.clients.compressor.v2.schemas import (
+    RequestCreateModel,
+    RequestUploadModel,
+    RequestValidateModel,
+    UploadFile,
+    ModelBase,
+    Options,
+    RecommendationOptions,
+    ResponseCompressionItem,
+    ResponseCompression,
+    RequestAvailableLayers,
+    ResponseSelectMethod,
+    RequestAutomaticCompressionParams,
+    RequestCreateCompression,
+    RequestCreateRecommendation,
+    RequestUpdateCompression,
+)
+from netspresso.clients.launcher import launcher_client_v2
+from netspresso.compressor.utils.file import read_file_bytes
+from netspresso.compressor.utils.onnx import export_onnx
+from netspresso.enums import CompressionMethod, Framework, Module, RecommendationMethod, ServiceCredit, Status, TaskType
+from netspresso.utils import FileHandler
+from netspresso.utils.metadata import MetadataHandler
+from netspresso.metadata.compressor import CompressorMetadata
+
+
+class CompressorV2:
+    def __init__(self, token_handler: TokenHandler) -> None:
+        """Initialize the Compressor."""
+
+        self.token_handler = token_handler
+
+    def check_credit_balance(self, service_credit):
```

Comment:
P4. What do you think about using the [existing function](https://github.com/Nota-NetsPresso/PyNetsPresso/blob/235-sprint-refactor-netspresso-architecture-and-integrate-new-api-client-for-on-prem-server/netspresso/utils/credit.py#L4)?